driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -2145,6 +2155,45 @@ public String toString() {
         }
     }
 
+    private static class VectorSearchBson<T> implements Bson {
```

Comment:
We don't need a type parameter here on in the constructor, and can use `Object` instead without losing any guarantees or introducing any inconvenience.